Updated "Fhir Data Pipes" concepts documentation (#1242)
google/fhir-data-pipes: doc/docs/concepts/views.md

## Overview

The heavily nested nature of FHIR resources and the
_[Parquet-on-FHIR schema](#parquet-on-fhir-schema)_
requires complex SQL queries that
can make them difficult to work with for analytics use cases. A common approach
to address this is to flatten the data into a set of views (virtual or
materialized) which can then be queried using simpler SQL statements.

## Parquet-on-FHIR schema

> The output of the _FHIR Data Pipes ETL Pipelines_ is conversion of raw FHIR 

Reviewer: ```suggestion
The _FHIR Data Pipes ETL Pipelines_ convert raw FHIR 
```

resources to a _Parquet-on-FHIR_ schema representation. This takes place for

> each resource type and follows the [FHIR Data Pipes Schema Mapping Rules](https://github.com/google/fhir-data-pipes/blob/master/doc/schema.md){:.external}.

Reviewer: ```suggestion
each resource type and follows the [Schema Mapping Rules](https://github.com/google/fhir-data-pipes/blob/master/doc/schema.md).
```

> The generated columnar Parquet files provide the 'base data warehouse' that can

Reviewer: ```suggestion
The generated columnar Parquet files provide the *"base data warehouse"* that can
```

## Output Data Formats

> Applying the FHIR ViewDefinition resources to the "base dwh" will generate materialized views which represent a "constrained" set of data to be used for downstream analytics applications (such as dashboards or reporting). 

Reviewer: nit1: please fix the style, e.g., break the lines at 80 chars.

nit2: consider adding a reference to the config param for enabling this, i.e., [viewDefinitionsDir](https://github.com/google/fhir-data-pipes/blob/ffecd7c7bf23d86bee0c6dde0dd5d549038f769d/pipelines/controller/config/application.yaml#L166C3-L166C21)

Author: SGTM. Can you fix/update this?

Reviewer: Done.

These can be outputted in any tabular format with current support for Database tables and Parquet files.

### Conversion to Database tables

> The resulting database tables can be loaded into a commonly used relational database management system such as [PostgresSQL](https://www.postgresql.org/) or [MySQL](https://www.mysql.com/).

Reviewer: nit: Consider adding a reference to the config param that enables this, i.e., [sinkDbConfigPath](https://github.com/google/fhir-data-pipes/blob/ffecd7c7bf23d86bee0c6dde0dd5d549038f769d/pipelines/controller/config/application.yaml#L173).

Author: SGTM. Can you fix/update this?

Reviewer: Done.
